third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1115,6 +1166,16 @@ fn check_ability(has_ability: bool) -> PartialVMResult<()> {
     }
 }
 
+impl AccessSpecifierEnv for Frame {
+    fn eval_address_specifier_function(
+        &self,
+        fun: AddressSpecifierFunction,
+        local: LocalIndex,
+    ) -> PartialVMResult<AccountAddress> {
+        fun.eval(self.locals.copy_loc(local as usize)?)
```

Comment:
I think they always must be copyable, either directly or via a reference. This is because they will be applied to parameters of transactions and cannot consume those.